### ISSUE TITLE
about the results

### ISSUE THREAD

**Issue by Winnie202** [llmpass/RSTT#9]
Excuse me, does frame insertion have a great impact on the output? Because my test results will have obvious flicker problems:For example, in the following output, the results of singular images are obviously better, and the results of even images are blurry
![0155](https://user-images.githubusercontent.com/86768013/209535911-ea32125a-f11d-4c4c-8b26-e8724aeafc49.png)
![0156](https://user-images.githubusercontent.com/86768013/209535930-f29bab9e-58bd-4706-adc0-a23d74be2324.png)
![0157](https://user-images.githubusercontent.com/86768013/209535939-c2ebfdcc-5063-4a4a-930a-15d31ae1084e.png)
![0158](https://user-images.githubusercontent.com/86768013/209535942-a4bd5425-050e-4ed0-811b-97ce5d6ec17a.png)

**Comment by llmpass** [comment]
Do you see the same issue on your training set?

**Comment by Winnie202** [comment]
> Do you see the same issue on your training set?
> 

No, the training set is normal

**Comment by llmpass** [comment]
> > Do you see the same issue on your training set?
> 
> No, the training set is normal

Then, it's an overfitting problem.

**Comment by Winnie202** [comment]
> > > Do you see the same issue on your training set?
> > 
> > 
> > No, the training set is normal
> 
> Then, it's an overfitting problem.

I have added training set data, but it has not been solved. Is there any way?

**Comment by llmpass** [comment]
> > > > Do you see the same issue on your training set?
> > > 
> > > 
> > > No, the training set is normal
> > 
> > 
> > Then, it's an overfitting problem.
> 
> I have added training set data, but it has not been solved. Is there any way?

sorry, I have no idea.

**Comment by Winnie202** [comment]
> > > > > Do you see the same issue on your training set?
> > > > 
> > > > 
> > > > No, the training set is normal
> > > 
> > > 
> > > Then, it's an overfitting problem.
> > 
> > 
> > I have added training set data, but it has not been solved. Is there any way?
> 
> sorry, I have no idea.

Thank you for your answer. If i adjust the learning rate and other relevant parameters, do you think it will help

**Comment by llmpass** [comment]
No, I don't think so, since the result on your training set is good enough.

**Comment by Winnie202** [comment]
> No, I don't think so, since the result on your training set is good enough.
> 
> Do you think this bad result is related to use interpolation for queries

**Comment by Winnie202** [comment]
> No, I don't think so, since the result on your training set is good enough.
> 
>  if the result on  training set has same issue,how to solve it
